plugins/Denizen/scripts/discord/bridge.dsc

```diff
@@ -10,3 +10,9 @@ discord_bridge:
         - discordmessage id:bot channel:1069156072197210112 "<&gt> *<player.name> joined*"
         after player quit:
         - discordmessage id:bot channel:1069156072197210112 "<&gt> *<player.name> left*"
+        after custom event id:forbidden_word_created:
+        - discordmessage id:bot channel:1069156072197210112 "<&gt> *'<context.word>' is now a **forbidden word!***"
+        after custom event id:forbidden_words_used:
+        - discordmessage id:bot channel:1069156072197210112 "`<player.name>`: *used forbidden word(s): **<context.words.formatted>***"
```

Comment:
Would it be possible in your PR to also make the word pluralize (real word? unsure) in the discord chat? it's not like important or anything but in the mc chat it says "words", but in the discord chat it just does "word(s)"

Reply:
make em use another diamond

Reply:
> make em use another diamond

Not sure what you mean?

Reply:
think it would me more fair to use diamonds for the plurals too

Reply:
This is in the context of formatting the forbidden words players have used - you may be misinterpreting something

Reply:
oh mb